src/index.test.ts

```diff
@@ -1,15 +1,17 @@
Input"}],"outputs":[{"to":"1C1tuBodLqmJcy7RH5oQHeBEhGvHhoccSA","satoshis":234423,"scripts":[{"address":"1C1tuBodLqmJcy7RH5oQHeBEhGvHhoccSA","satoshis":234423,"script":"76a91478d4b515b0e9de6fa483889eea9e1285630247ae88ac","typename":"ScriptOutput"}],"paymailP4":null,"typename":"TransactionOutput"},{"to":"18C3n8CDgtxc62b56V1sNorR7wAToF9jLH","satoshis":37018825,"scripts":[{"address":"18C3n8CDgtxc62b56V1sNorR7wAToF9jLH","satoshis":37018825,"script":"76a9144edfc667fab03d524fd96c487262369b12d22a3188ac","typename":"ScriptOutput"}],"paymailP4":null,"typename":"TransactionOutput"},{"to":"18Y9SoziUyrzo2JoNnfwrQhFamvpPkNfhM","satoshis":27051980,"scripts":[{"address":"18Y9SoziUyrzo2JoNnfwrQhFamvpPkNfhM","satoshis":27051980,"script":"76a91452acd3287e875e14e4c9dba8cb0a33e7b1cb3fdd88ac","typename":"ScriptOutput"}],"paymailP4":null,"typename":"TransactionOutput"},{"to":"1K8auRR89TN7AJW1JrgcafBmXhdWXvTxpz","satoshis":35597248,"scripts":[{"address":"1K8auRR89TN7AJW1JrgcafBmXhdWXvTxpz","satoshis":35597248,"script":"76a914c6e17a315b6766b0244e9d54667202730f8dc20888ac","typename":"ScriptOutput"}],"paymailP4":null,"typename":"TransactionOutput"}],"changeDestinations":[{"address":"18C3n8CDgtxc62b56V1sNorR7wAToF9jLH","chain":1,"num":112,"lockingScript":"76a9144edfc667fab03d524fd96c487262369b12d22a3188ac","draftId":"8059aece83eb455c92965c331f406fa636e75985455c446402e15566cecaba44","typename":"Destination"},{"address":"18Y9SoziUyrzo2JoNnfwrQhFamvpPkNfhM","chain":1,"num":113,"lockingScript":"76a91452acd3287e875e14e4c9dba8cb0a33e7b1cb3fdd88ac","draftId":"8059aece83eb455c92965c331f406fa636e75985455c446402e15566cecaba44","typename":"Destination"},{"address":"1K8auRR89TN7AJW1JrgcafBmXhdWXvTxpz","chain":1,"num":114,"lockingScript":"76a914c6e17a315b6766b0244e9d54667202730f8dc20888ac","draftId":"8059aece83eb455c92965c331f406fa636e75985455c446402e15566cecaba44","typename":"Destination"}],"changeSatoshis":99668053,"fee":615,"typename":"TransactionConfig"},"status":"draft","expiresAt":"2022-03-04T14:26:34.755995Z","hex":"0100000008752e0a1765a511378738be481f917b5722aad3ba71e5bd1944de38353b88c3570100000000ffffffff243ded05e39b3f635768e1679a3b74dbb0f50e824308a94e6696fb5e63100e480100000000ffffffff01d9e9e83491322d81288db7fd57b9a4e7426e07d814b73194527f64f1fabf760100000000ffffffff01d9e9e83491322d81288db7fd57b9a4e7426e07d814b73194527f64f1fabf760300000000ffffffff76b8311575ce1366b55b3c930857bdb6157924bc8459150e0d0c5a0fcc209a810300000000ffffffff148b4157411359202afd83859e9857748f06091370c8871e18dd9b6747d1430f0200000000ffffffffb3b5279da2536ce8a0f4312bbc153c182ca0fdf91192f662acf66f107e71ca6a0000000000ffffffffb3b5279da2536ce8a0f4312bbc153c182ca0fdf91192f662acf66f107e71ca6a0300000000ffffffff04b7930300000000001976a91478d4b515b0e9de6fa483889eea9e1285630247ae88acc9dc3402000000001976a9144edfc667fab03d524fd96c487262369b12d22a3188acccc79c01000000001976a91452acd3287e875e14e4c9dba8cb0a33e7b1cb3fdd88acc02b1f02000000001976a914c6e17a315b6766b0244e9d54667202730f8dc20888ac00000000","typename":"DraftTransaction"}`;
```

Comment:
I don't know, why you change those things?

Reply:
I changed them, because it had snake_cased fixtures instead of camelCased so the test didn't pass.